cost-analyzer/values.yaml

```diff
@@ -405,6 +405,15 @@ kubecostFrontend:
 #    fqdn: kubecost-api.kubecost.svc.cluster.local:9001
 #  model:
 #    fqdn: kubecost-model.kubecost.svc.cluster.local:9003
+#  forecasting:
+#    fqdn: kubecost-forcasting.kubecost.svc.cluster.local:5000
+#  aggregator:
+#    fqdn: kubecost-aggregator.kubecost.svc.cluster.local:9004
+#  cloudCost:
+#    fqdn: kubecost-cloud-cost.kubecost.svc.cluster.local:9005
+#  multi_cluster_diagnostics:
```

Comment:
Camel case convention is preferred here (multiClusterDiagnostics).

Reply:
Updated.